backend/de.metas.externalreference/src/main/sql/postgresql/system/80-de.metas.externalreference/5658230_sys_gh13685_ExternalReference_IsReadOnlyInMetasfresh_ErrorMessage.sql

```diff
@@ -0,0 +1,54 @@
+-- Value: externalReferenceReadOnlyInMetasfresh
+-- 2022-09-30T13:15:38.066Z
+INSERT INTO AD_Message (AD_Client_ID,AD_Message_ID,AD_Org_ID,Created,CreatedBy,EntityType,IsActive,MsgText,MsgType,Updated,UpdatedBy,Value) VALUES (0,545160,0,TO_TIMESTAMP('2022-09-30 16:15:37','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','Document {0} cannot be deleted since it is externally referenced as read only in metasfresh.','I',TO_TIMESTAMP('2022-09-30 16:15:37','YYYY-MM-DD HH24:MI:SS'),100,'externalReferenceReadOnlyInMetasfresh')
```

Comment:
pls give it the proper message key, e.g. something like CannotDeleteExternalReferenceReadOnlyInMetasfresh